### PR TITLE
Antegony todo

### PR DESCRIPTION
**Pull request by muchness**
http://cryptic-oasis-9126.herokuapp.com/

### CODE REVIEW

**Comment by muchness** [comment]
fierce-sea-7183.herokuapp.com
@RyanHedges RyanHedges herokuapp

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 *.sqlite3
 *.swp
+.env
```
Comment (supertopher) [inline .gitignore:3]: talk to me tomorrow about .gitignore global files... or just check out .gitconfig and .gitignore_global from my repo here https://github.com/supertopher/dotfiles

Comment (zspencer) [inline .gitignore:3]: Many repositories have multiple .env files:
1. `.env` which includes the publicly fine to share values
2. `.env.development`/`.env.test` which contains the private values, and is ignored
3. `.env.example` which includes the names of the environment variables with bogus data and comments on how to get the real data.

File: todo_app.rb [hunk 2]
```diff
@@ -7,11 +7,18 @@
 
 
 # This loads environment variables from the .env file
-require 'dotenv'
-Dotenv.load
+# require 'dotenv'
+# Dotenv.load
```
Comment (zspencer) [inline todo_app.rb:11]: Why are these commented out?

File: views/index.erb [hunk 3]
```diff
@@ -1 +1,12 @@
...
+  <input type="submit" value="Create Todo">
+</form>
+
+<h1>Here's all the things to do:</h1>
+
+<!-- begin -->
+<% Todo.all.each do |task|%>
+  <%= task.task %><br>
+<% end %>
+<!-- rescue -->
```
Comment (zspencer) [inline views/index.erb:12]: I would recommend exposing an instance variable in your route; like so:

```
get '/' do
  @todos = Todo.all
   # ... the rest of your route
end
```

Then modifying your erb file to reference the instance variable:

```
<% @todos.each do |task| %>
  <%= task.task %>
<% end %>
```

This makes it so your views don't know about your underlying class structure.

File: todo_app.rb [hunk 4]
```diff
@@ -7,11 +7,18 @@
...
 
 get '/' do
   erb :index
 end
+
+
+post '/' do
+  Todo.create(:task => params[:"New Todo"] )
+  erb :index
+  # params.inspect
```
Comment (zspencer) [inline todo_app.rb:23]: I strongly recommend deleting commented out code that was used for debugging.

**Comment by RyanHedges** [comment]
Hey! 👋 

I'm closing this PR. I'm now using [Pullwalla](https://pullwalla.com/) to see statuses of all my pull requests across all projects. This PR is stale and creating noise. Thanks for the memories! 🤟

**Pull request closed without merging**